indigo/distribution.yaml

```diff
@@ -466,6 +466,31 @@ repositories:
       url: https://github.com/GT-RAIL/async_web_server_cpp.git
       version: develop
     status: maintained
+  aubo_robot:
+    doc:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: 0.1.0
+    release:
+      packages:
+      - aubo_control
+      - aubo_description
+      - aubo_driver
+      - aubo_gazebo
+      - aubo_i5_moveit_config
+      - aubo_kinematics
+      - aubo_msgs
+      - aubo_robot
+      - aubo_trajectory
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/auboliuxin/aubo_robot-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: 0.1.0
```

Comment:
The source entry needs to be a branch name, so in your case it should be `master`